cscs-checks/apps/vasp/vasp_check.py

```diff
@@ -8,16 +8,19 @@
 
 
 class VASPCheck(rfm.RunOnlyRegressionTest):
-    keep_files = ['OUTCAR']
-    maintainers = ['LM']
-    tags = {'scs'}
-    strict_check = False
+    scale = parameter(['small', 'large'])
```

Comment:
I would be more explicit here and name this parameter as `num_nodes`. Check how we do it with the Gromacs test. The problem with this approach is that the variant has different meaning for different systems which can be hard to follow.

Reply:
I set `num_nodes = parameter([6, 16], loggable=True)`